modules/highlight/highlight_test.go

```diff
@@ -78,10 +78,11 @@ c=2
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			out, err := File(tt.name, "", []byte(tt.code))
+			out, lexerName, err := File(tt.name, "", []byte(tt.code))
 			assert.NoError(t, err)
 			expected := strings.Join(tt.want, "\n")
 			actual := strings.Join(out, "\n")
+			assert.NotEmpty(t, lexerName)
```

Comment:
Should we perhaps enhance the test to test if the expected lexer has been found?

Reply:
Hmm yes, we could add that.

Reply:
Done